samcli/lib/build/app_builder.py

```diff
@@ -216,20 +216,27 @@ def update_template(template_dict: Dict, original_template_path: str, built_arti
 
             resource_type = resource.get("Type")
             properties = resource.setdefault("Properties", {})
+            package_type = properties.get("PackageType", ZIP)
 
-            if resource_type == SamBaseProvider.SERVERLESS_FUNCTION and properties.get("PackageType", ZIP) == ZIP:
-                properties["CodeUri"] = store_path
+            if resource_type == SamBaseProvider.SERVERLESS_FUNCTION and package_type == ZIP:
+                if "InlineCode" not in properties or properties["InlineCode"] is None:
```

Comment:
Why do we need to check for InlineCode here? On line 201, we check to see if the function was built, so assuming that we don't build any functions with InlineCode, this is would never execute right?

Reply:
@jfuss you're right, other changes are sufficient, verified that this check is unnecessary